client/reader/following-edit/style.scss

```diff
@@ -146,7 +146,7 @@
 
 	.card.is-search-result {
 		position: absolute;
-			top: 51px;
+		top: 51px;
```

Comment:
These are on purpose :) 
We indent positioning rules for a reason I never understood. But we do.